src/Articles/tests/redux/reducers/SearchArticlesReducers.test.jsx

```diff
@@ -0,0 +1,55 @@
+import SearchArticleReducer from '../../../redux/reducers/SearchArticleReducer';
+import { SEARCH_ARTICLE_SUCCESS, SEARCH_ARTICLE_FAILED, START_SEARCHING_ARTICLES } from '../../../redux/types';
+
+const initialState = {};
+
+const searchAction = {
+  type: SEARCH_ARTICLE_SUCCESS,
+  payload: [{ articles: ['1'] }],
+  message: 'Articles successfully retrieved',
+  status: true,
+  is_fetching: true,
+};
+
+const searchFailedAction = {
+  type: SEARCH_ARTICLE_FAILED,
+  message: undefined,
+  items: [],
+  status: false,
+  is_fetching: false,
```

Comment:
@alexxsanya please do change all incidents of this in your task.

Reply:
@neelxie thanks for this feedback. I am working on it.

Reply:
@neelxie I am done working on this.